Fulfillment/Integration/01_ManageConnections.md

```diff
@@ -0,0 +1,143 @@
+
+#### Prerequisites
+
+At least one driver has been installed.
+
+#### Procedure
+
+*Fulfillment > Settings > Tab CONNECTIONS*
+
+![Connection](../../Assets/Screenshots/Fulfillment/Settings/Connections/Connections.png "[Connection]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+    The *Create connection* view is displayed.
+
+    ![Create connection](../../Assets/Screenshots/Fulfillment/Settings/Connections/CreateConnection.png "[Create connection]")
+
+2. Enter a name for the connection in the *Name* field.
+
+3. Click the *Driver* drop-down list and select the desired driver. All available drivers are displayed in the list.   
+
+    > [Info] Drivers must be purchased from the main account and then be installed separately in each sandbox or account where you want to use it.   
+
+    The *Credentials* section is displayed below the drop-down list.
+
+    ![Credentials](../../Assets/Screenshots/Fulfillment/Settings/Connections/Credentials.png "[Credentials]")
+
+    > [Info] Depending on the selected driver, the fields in the *Credentials* section differ.
+
+
+4. Enter the required data in the fields in the *Credentials* section.
+
+5. Click the [SAVE] button.   
+    The connection will be established. The *Checking credentials...* notice is displayed. After a few seconds, the *Loading data...* notice is displayed.
+
+    ![Checking credentials](../../Assets/Screenshots/Fulfillment/Settings/Connections/CheckingCredentials.png "[Checking credentials]")
+
+    The *Create connection* view is automatically closed when the connection has been established. The *Edit connection* view of the newly created connection is displayed. The *Settings* tab is selected.
+
+    ![Edit connection settings](../../Assets/Screenshots/Fulfillment/Settings/Connections/EditConnectionSettings.png "[Edit connection settings]")
+
+    > [Info] By default, all connections that have been established are first inactive and therefore must be enabled after the connection configuration, see [Enable a connection](#enable-a-connection).
+
+
+## Edit a connection
+
+Edit a connection to adjust any changed credentials, the connection name or further settings. The driver selection is read-only and cannot be edited.
+
+#### Prerequisites
+
+At least one connection has been created, see [Create a connection](#create-a-connection).
+
+#### Procedure
+
+*Fulfillment > Settings > Tab CONNECTIONS*
+
+![Connections](../../Assets/Screenshots/Fulfillment/Settings/Connections/Connections.png "[Connections]")
+
+1. Click the connection you want to edit in the list of connections.   
+    The *Edit connection* view is displayed.
+
+    > [Info] Depending on the driver of the connection, the *Credentials* and the *Settings* tab are displayed in the *Edit connection* view.
```

Comment:
Wenn alle Connections in Fulfillment Credentials und Settings haben, kann die Info rausfliegen.

Reply:
nein nicht alle haben extra settings